include/jwt-cpp/traits/danielaparker-jsoncons/traits.h

```diff
@@ -61,7 +61,12 @@ namespace jwt {
 					return 0;
 				}
 			};
-			using array_type = json::array;
+			class array_type : public json::array {
+			public:
+				value_type& front() { return this->operator[](0U); }
```

Comment:
Do we need the none const version?


We only ever call it with const from `as_string`
https://github.com/Thalhammer/jwt-cpp/blob/feaca3cc8541e49b3508fdd07abe83851f858c91/include/jwt-cpp/traits/open-source-parsers-jsoncpp/traits.h#L108

Reply:
`jsoncpp` supported both, so I thought no harm to add both and copied them to `jsoncons` as well.
I'll remove them.